app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/DisplayOptionsView.java

```diff
@@ -193,6 +200,10 @@ public boolean isEditing() {
         setAutoplay(SettingsStore.AUTOPLAY_ENABLED, true);
         setCurvedDisplay(false, true);
 
+        if (mBinding.startwithPassthroughSwitch.isChecked() != SettingsStore.START_WITH_PASSTHROUGH_DEFAULT) {
+            setStartwithPassthrough(SettingsStore.START_WITH_PASSTHROUGH_DEFAULT);
+        }
```

Comment:
I've noticed that the change I'm proposing above will break things like this (as the default for the A3 won't be the default we want).

Reply:
Then this will not be broken